spec/factories/conferences.rb

```diff
@@ -7,5 +7,12 @@
     twitter_username "twiiter"
     image_url "image_url"
     website "foo.bar"
+
+    factory :ruby_conference, class: 'RubyConference' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.